gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -457,6 +462,15 @@ private void submitJob(DagNode<JobExecutionPlan> dagNode) {
       JobSpec jobSpec = DagManagerUtils.getJobSpec(dagNode);
       Map<String, String> jobMetadata = TimingEventUtils.getJobMetadata(Maps.newHashMap(), jobExecutionPlan);
 
+      // Increment submission attempt
```

Comment:
Should this blocked be part of `TimingEventUtils.getJobMetadata` ?

Reply:
I don't think the incrementing logic should be in there because it could be called at a time other than job submission. But I moved the logic for updating the map to there.